### DISCUSSION TITLE
How do I know if some software requires cache?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1077] [Q&A]
If some software doesn't work but it doesn't tell me that it has some input/output error, how do I know whether it fails because I don't have cache in the mergerfs folder, after using the `cache.files=off` option?

**Comment by trapexit** [comment]
If it uses mmap it's likely a critical failure when it fails.

**Comment by trapexit** [comment]
It's not that software "require" cache. It's that when caching is disabled so is certain usages of mmap. If software use mmap in those ways it will fail. It will receive errors.
